### ISSUE TITLE
CI not running all unit tests

### ISSUE THREAD

**Issue by aanm** [cilium/cilium#4689]
It seems https://github.com/cilium/cilium/pull/4670 broke the unit tests and the CI didn't catch it. It was later fixed by https://github.com/cilium/cilium/pull/4685

**Comment by ianvernon** [comment]
The CI from https://github.com/cilium/cilium/pull/4670 did run the unit test which got broken.
Build link: https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/4495/execution/node/55/log/?consoleFull

Output from unit test which was fixed in #4685 : 

```
09:44:51 PASS: <autogenerated>:1: DaemonEtcdSuite.TestMinimumWorkerThreadsIsSet	0.000s
...
09:44:51 PASS: <autogenerated>:1: DaemonConsulSuite.TestMinimumWorkerThreadsIsSet	0.000s
```

#4685 was filed when I ran the tests in the developer VM with `NFS=1 ./contrib/vagrant/start.sh`. The unit tests in the CI run directly on the host via Docker Compose, which I suspect uses the number of CPUs on the packet.net instance to check the minimum number of CPUs.

**Comment by ianvernon** [comment]
@aanm can this be closed?
